worldedit-core/src/main/java/com/fastasyncworldedit/core/configuration/Settings.java

```diff
@@ -625,16 +626,10 @@ public FaweLimit getLimit(Actor actor) {
         public boolean OTHER = false;
 
         @Comment({
-                "Allow blocks placed by WorldEdit to tick. This could cause the big lags.",
+                "Allow fluids placed by FAWE to tick (flow). This could cause the big lags.",
```

Comment:
Maybe worth to add a note that ticked blocks are not captured in history and therefore not available for undo/redo.